### PR TITLE
API-Docs - Daily Menu

### PR DESCRIPTION
**Pull request by hungbkdn**

### CODE REVIEW

File: docs/api-references/daily-menus.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
+## Daily Menu - API
+
+### `GET` List Daily Menu
+```
+/daily-menus
+```
+Get all daily menu list
+#### Request Header
```
Comment (ghost) [inline docs/api-references/daily-menus.md]: **Request Header** is common, you don't need to repeat every API.

File: docs/api-references/daily-menus.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+| Key | Value |
+|---|---|
+|Accept|application/json
+
+#### Sample Response
+```json
+{
+    "current_page": 1,
```
Comment (ghost) [inline docs/api-references/daily-menus.md]: Why `current_page` display before date? Laravel's pagination return like this?

File: docs/api-references/daily-menus.md [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+	"success": true
+}
+```
+
+### `GET` Daily Menu Detail
+```
+/daily-menus/{date}
+```
+Get all food and its information in daily menu by date
+#### Request Header
```
Comment (ghost) [inline docs/api-references/daily-menus.md]: **Request Header** is common, you don't need to repeat every API.

File: docs/api-references/daily-menus.md [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+| Key | Value |
+|---|---|
+|Accept|application/json
+
+
+#### Sample Response
+```json
+{
+    "data": {
+        "current_page": 1,
```
Comment (ghost) [inline docs/api-references/daily-menus.md]: Why `current_page` display before date? Laravel's pagination return like this?

File: docs/api-references/daily-menus.md [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+#### Request Header
+| Key | Value |
+|---|---|
+|Accept|application/json
+
+
+#### Sample Response
+```json
+{
+    "data": {
```
Comment (ghost) [inline docs/api-references/daily-menus.md]: Why this result has `data` in the first level?

**Comment by ghost** [comment]
LGTM

**Comment by hungbkdn** [comment]
I have added Request Header for each API, please check it again!
Thank you so much

**Pull request merged**